solarwinds_apm/apm_constants.py

```diff
@@ -35,6 +35,20 @@
     INTL_SWO_BAGGAGE_PROPAGATOR,
     INTL_SWO_PROPAGATOR,
 ]
+INTL_SWO_DEFAULT_RESOURCE_DETECTORS = [
+    "process",
+    "os",
+    "host",
+    "aws_ec2",
+    "aws_ecs",
+    "aws_eks",
+    "aws_lambda",
+    "azure_app_service",
+    "azure_functions",
+    "azure_vm",
+    "k8s",
+    "uams",
```

Comment:
IIUC, `uams` and all other resource detectors in this list will be loaded by calls to Otel SDK `Resource.create` e.g. by Configurator at startup. For an APM-autoinstrumented service where uams client hasn't been installed, does `Resource.create` quietly not set the uams attributes? Same for if service is not in k8s.

Reply:
Yes, it prints a debug message saying that the uamsclientid file is not found and cannot get anything from uamsclient API.